plugins/inline-row-separators/test/inline_row_separators_test.mocha.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+require('../src/inline_row_separators_renderer');
+const chai = require('chai');
+const Blockly = require('blockly');
+const jsdomGlobal = require('jsdom-global');
+const {overrideOldBlockDefinitions} = require('../src/index');
+
+overrideOldBlockDefinitions();
+
+const assert = chai.assert;
+
+suite('inlineRowSeparates', function() {
+  setup(function() {
+    this.jsdomCleanup =
+        jsdomGlobal('<!DOCTYPE html><div id="blocklyDiv"></div>');
+    this.workspace = Blockly.inject('blocklyDiv', {
+      renderer: 'inline-row-separators',
+    });
+    this.renderer = this.workspace.getRenderer();
+  });
+
+  teardown(function() {
+    this.workspace.dispose();
+    this.jsdomCleanup();
+  });
+
+  test('a text join block should start new rows for value inputs', function() {
```

Comment:
This test is testing a lot of things, `overrideOldBlockDefinitions`, the new `text_join` definition, and `should StartNewRow_`. This will make it hard to figure out what exactly is wrong if any of those things break in the future.

Can you separate those into different tests? In particular I think `shouldStartNewRow_` should be tested separately from the other bits.

Reply:
I've split it into a bunch of tests and added a new one for the ability to add row separators to custom renderers.